BLSQ/iaso
# POLIO-1803: adapt supplychain to bopv/nopv new vaccine

What problem is this PR solving? Explain here in one sentence.

Related JIRA tickets : POLIO-1803

## Self proofreading checklist


## Doc

Tell us where the doc can be found (docs folder, wiki, in the code...).

## Changes

- Add model methods to return single vaccines on Round and Campaign:
   - Includes sub-activities scopes
- Show individual vaccines for bOPV & nOPV2 campaigns in VRF + supplychain & vaccine stock vaccine filters


## How to test

Create a campaign with a nOPV2 & bOPV scope
Add a sub-activity with mOPV

Go to supplychain
create a VRF for the campaign: the vaccine dropdown should show all 3 vaccines

Check supplychain and vaccine stock vaccine filters: "nOPV2 & bOPV" should not appear (only individual vaccines)

## Print screen / video

Uploading Screen Recording 2024-12-19 at 14.46.17.mov…




## Notes

merges in to POLIO-1762-nOPV2-bOPV-cVDPV2-cVDPV3

## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
